### PR TITLE
Newsweek homepage clone.

### PR DESCRIPTION
**Pull request by dipbazz**
This project is about building a [news week](https://www.newsweek.com/) home page clone.

The features of this project are listed as below:
1. The website is completely responsive with different breakpoint provided by bootstrap:
    - Below 575px for mobile screen
    - In between 575px and 768px larger mobile screen.
    - In between 768px and 992px for medium screen (tablet).
    - In between 992px and 1200px for laptop and desktop.
    - Above 1200px for large screen.

2. The site contains information:
    - Header with site navigation links.
    - Top Stroy
    - Featured stories.
    - The debate.
    - Opinions.
    - More stories.
    - Different articles.
    - In the magazine.
    - Editor's Pick
    - Subscribe and newsletter.
    - Footer with other useful links.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,79 +1,87 @@
...
 
-👤 **Author1**
+👤 **Yoko Saka / Vicky**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/linkedinhandle)
+    - GitHub: [@yocosaka](https://github.com/yocosaka)
+    - Twitter: [@yocosaka](https://twitter.com/yocosaka)
+    - LinkedIn: [Yoko Saka](https://www.linkedin.com/in/yokosaka)
```
Comment (javitocor) [inline README.md]: Kindly, make this contact look like the other one, it is more professional

**Review by javitocor — CHANGES_REQUESTED** [review]
# STATUS CHANGES REQUIRED ♻️ 

Hello team!
Great job with that project! I am impressed! 👍 
You really master bootstrap! 🥇 
And only used less than 300 lines of CSS 👏 
Nice set up of the linters! 🟢 
Professional readme file! 💯 

However, we need to make some small changes before approving this project 💪 
Kindly, check my comments below 👇  apply the changes and resubmit your project for a new code review ⏩ 

## Mandatory Changes
-  The navbar must be fixed/sticky for all screen sizes, kindly make it happen
- Screen sizes smaller than 1024px:
   - The US, business, sports...sections are missing, please add it to this screen size
   - Kindly make the signup section, equal to the original, it shouldn't take 100% width
original:
Capture
your project:
Capture1
   - Kindly fix the first image in the editors pick section, must be all items equal
Capture2
   - Fix the image in the newsletter section
original
Capture3
your project:
Capture4
- Screens below 768px
   - Kindly fix the footer, the social icons must be under the letters
original
Capture5
your project
Capture6
   - The US, business, sports...sections are missing, please add it to this screen size

**Comment by dipbazz** [comment]
Thank you for your code review @javitocor. We really appreciated and almost fixed everything you have mentioned for a change but we couldn't achieve the navbar sticky by using only CSS. So should we use javascript to achieve that sticky behavior or can we skip for now?

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
### Request changes 😳
This is an amazing project. You guys worked really amazing on this project but there is still some room for improvement. 
_It's okay. We are here to learn. Don't get discouraged. Work even harder.Let's make good mistakes next time. You are almost there.😄_
**Please have a look at the suggestions to get more understanding.**
### General

- [x] All **alt** attributes should have a short and meaningful image description.
- [x] Please use less CSS and more bootstrap for this project. You can easily achieve some tasks like display, hight, width flex, etc from bootstrap. Please work on it and change CSS to bootstrap. 
image

- [x] Your license link is broken in your Readme. Please add a license file to your project. 
 
### Large Screen


- [x] The top navbar should be sticky. It decreases its width and removes the date and weather on scrolling. But on your page navbar is not sticky. You can learn about sticky navbar in bootstrap [here](https://www.w3schools.com/howto/howto_js_navbar_sticky.asp) and here is a [video ](https://www.youtube.com/watch?v=qnV1wbcKymM). 

**Original** 

Screenshot (20)

**Yours Clone**

Screenshot (19)



- [x] **This is optional but highly recommended too**. On the original page there a sticky part.  This part makes the webpage more interesting and shows good knowledge and usage of bootstrap. But on your webpage, this page is missing. If you can Please work on it. 
chrome-capture (34)

- [x] On the original page, the newsletter page has the same height as the subscribe section. But on your page, that is a problem with it. Please work on it. 

**Yours Clone**

chrome-capture (36)

**Original Page**

chrome-capture (35)

- [x] On your clone Newsweek Logo has more width as compared to the original page. And the social icons are not correctly aligned. 

### Medium(Tablet) Screen

- [x] The top navbar should be sticky on medium screen too. Below there are the images attached. Please have a look here. 

**Yours Clone**

chrome-capture (41)

**Original** 

chrome-capture (40)

- [x] When scrolling down, a white space appears which is not present on the original page. My recommendation is to make it as present on the original page. 

chrome-capture (42)

- [x] On the original page the image has lesser height than the other images present on the left side. Please have a look below. You can remove this using bootstrap. Don't use CSS. 

**Clone**

chrome-capture (44)

**Original** 

chrome-capture (43)

### Small Screen

- [x] Top navbar is also sticky for small screens. Please work on it. You can achieve this goal by bootstrap purely. 

- [x] on small screens, the whole footer is aligned vertically at the same position but in your clone, there is a margin around the links.  Can you please fix it? 
chrome-capture (45)

- [x] On small screens this section should be in the center. 
chrome-capture (46)


**Please make the changes and request re-review for the given project**
Stay safe and give me a smile 😇

### Happy Coding
---------------------------------------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

## :wave: :wave: Hello @yocosaka and @dipbazz! Great work!

There are some issues that need to be addressed before we can move forward. Please review the list below.

## Required changes :wrench: 
### [Mandatory] :red_circle: 

- [x] MIT License link is broken.

- [x] All ```alt=""``` atributtes should have a short and meaningful image description.

- [x] Sections missing in tablet view: US, World, Business, Tech & science...
image

- [x] Element doesn't match original
image

- [x] Elements are not aligned to logos bottom
image

- [x] Same sections missing on mobile version

- [x] Subscribe section doesn't match original
image

- [x] Newsletter section doesn't match original
image

- [x] Footer doesn't match original
image

### [OPTIONAL]  :yellow_circle:  

- [x] Prefer HTML classes over ids. This helps reusability, and also keeps the code DRY.

- [x] Update the title of the website it is not a test anymore. :wink: 

- [x] Element doesn't match original in tablet view
image

### [Comments] :green_circle: 

- A great tool to test project and observe changes in the original website is the 
Browsers media query inspector. Please use it to verify the project looks as intended in different screen sizes.

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
## changes Requested
You have done a great job with this project but Please have a look below. 

- [x] In your readme you said to clone this project just clone it. Please remove that part. This will help you to present your project more professionally. 

- [x] Text is overflowing in small screens. Please work on it an submit a new request for review. 
95767505-75fd2580-0c7a-11eb-9990-f922a44463da
Please work on these two changes and submit a new request for review. 
**Regards,
Arslan Bisharat
slack: arslanbisharat**

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
### Changes Requested
Thank You for making some changes requested by me in the previous review. 
As requested by previous TSEs your website has text overflowing. Please work on it. In case you don't know how to fix that and need a 1:1 meeting with Full-TIme TSEs Please search on zendesk. 
**Regards,
Arslan Bisharat
Slack: arslanbisharat**

**Comment by dipbazz** [comment]
Thank you every TSEs for your reviews. We have done every change that was requested. We have fixed the text overflow on the subscribe section @arslanbisharat, we don't see the problem on our browser it's completely working and exactly the same as the original. We are confused about your concern about text overflow. If you could explain it clearly then it would help us to clear our doubts.

**Review by umairarshadbutt — APPROVED** [review]
# **Status: PROJECT APPROVED** :champagne:
Excellent job on this project guys. :fireworks: Keep it up.
##
### The [clone](https://yocosaka.github.io/newsweek-clone/) shows a good understanding and usage of Bootstrap. :100:
##
- The cloned page is rendering as it should and meets all the project specifications.
- The project has a descriptive README.md and is professionally done. Good job with that.
- Pull Request has a meaningful title and a short description.
- CSS is clean. No linter errors at this point.
- The repo has followed all the guidelines there are to follow (GitHub flow etc.)
**All the required tasks have been fulfilled in this project. I will go ahead and mark this as complete.**
- Congratulations on the splendid job @yocosaka and @dipbazz! :+1: 
- You can now merge to master and submit the project completion form :rocket: . I wish you all the very best in your next tasks :1st_place_medal:
##
**If you have any questions, let me know on slack at any time. I will be happy to help.**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Pull request merged**